modules/govuk/files/node/s_db_admin/reboot_elasticache.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+import argparse
+import boto3
+import logging
+
+
+def create_elasticache_client():
+  try:
+    logging.info("getting elasticache client")
+    client = boto3.client('elasticache')
+    return client
+  except Exception as e:
+    logging.error("error occurred when getting elasticache client {}".format(e.message))
+    raise
+
+def separate_cluster_and_node_ids(aggregated_cluster_node_ids):
```

Comment:
This looks like it just returns its argument (twice). Was it supposed to do something else?

Reply:
I will remove this unused function, it was just a relic of testing the api because the naming is so inconsistent with the aws gui.